api-spec/transactions-api.yaml

```diff
@@ -338,17 +335,28 @@ components:
               paymentToken: "paymentToken1"
               reason: "reason1"
               amount: 100
-              authToken: "authToken1"
             - rptId: "77777777777302012387654312385"
               paymentToken: "paymentToken2"
               reason: "reason2"
               amount: 100
-              authToken: "authToken2"
         status:
           $ref: "#/components/schemas/TransactionStatus"
+        amountTotal:
+          $ref: "#/components/schemas/AmountEuroCents"
+        feeTotal:
+          $ref: "#/components/schemas/AmountEuroCents"
+        origin:
+          description: cart origin
+          enum:
+            - IO
+            - CHECKOUT
```

Comment:
do you need to have `CHECKOUT` and `CHECKOUT_CART`?

Reply:
I found that values [here](https://pagopa.atlassian.net/wiki/spaces/I/pages/568099134/2022-10-26+-+Refinement+POST+transactions+gestione+carrello). Moreover they reflect the same values specified in commons package for transaction document.